workers/limiter/tests/index.test.ts

```diff
@@ -221,8 +223,11 @@ describe('Limiter worker', () => {
        * Assert
        */
       const result = await redisClient.sMembers('DisabledProjectsSet');
+      const updatedWorkspace = await workspaceCollection.findOne({ _id: workspace._id });
 
       expect(result).toContain(project._id.toString());
+      expect(updatedWorkspace.isBlocked).toBe(true);
+      expect(updatedWorkspace.blockedDate).toBeInstanceOf(Date);
```

Comment:
this test designed to check whether projects of workspace are blocked or not. I think, there should be a separate test for workspace blocking/unblocking